### ISSUE TITLE
Thanks

### ISSUE THREAD

**Issue by Exched** [yanickflyer/airlinemanager4bot#1]
Thank you for making this bot,I wanted to ask if you would provide a bot installation tutorial, I always get an error message 
File "/storage/emulated/0/Download/airlinemanager4bot/src/session.py", line 5, in auth
    "PHPSESSID":os.environ["PHP_SESSION"]
                ~~~~~~~~~~^^^^^^^^^^^^^^^
  File "<frozen os>", line 679, in __getitem__
KeyError: 'PHP_SESSION'

**Comment by yanickflyer** [comment]
Hello Exched,

Thanks for the interest in the bot.
I am not too sure if I will have a tutorial for this.

But basically the error you are getting is that it cannot find your php session id.

Taking a wild guess here, I assume you are running this on an android device?

If you look at the README, there is a screenshot on how to retrieve this PHP Session in Chrome.
Am not too sure how to retrieve that in the App Version of the game if at all possible.

Now that you have this php session ID, you add it to an environment variable called **PHP_SESSION**. I personnally use docker so passing an environment variable is straight forward. Not too sure how environment variables work in Android either.

So in short, the bot needs this **PHP_SESSION** to authenticate with the game. It's the only requirement in fact to run the bot.

**Comment by Exched** [comment]
I've copied from my PC browser to my Android, but it still doesn't work, is the format just PHP_SESSION=<php from browser>,or I don't need to add <>

**Comment by yanickflyer** [comment]
Difficult to say. Android is not my strong point.
Here's an article I have found on environment variables with Android:
https://developer.android.com/tools/variables

I wonder how are you running the script? Are using some kind terminal emulator? Or you developped an app on Android Studio?

**Comment by Exched** [comment]
I used termux and installed python on it, anyway thanks for helping, I'll try running it on my pc

**Comment by yanickflyer** [comment]
Try the following command before you run it:

export PHP_SESSION=**YOUR SESSION**
